domains/getsono.json

```diff
@@ -6,7 +6,8 @@
         },
     
         "record": {
-            "CNAME": "getsono.github.io"
+            "CNAME": "getsono.github.io",
+            "TXT": "google-site-verification=MFPLEonkdLiPBmou5SY2mk4qJQ2HC3jZC9cQ8poP3_o"
```

Comment:
You can't use CNAME with other records. (also for google site verification can't you do via html?)